adding e-span model

miuev opened the pull request:
e-span model added in canela/bin, with example energy input file in canela/examples

miuev committed:
adding e-span model

michael-cowan reviewed (changes requested):
I think you should convert this to a click script and then update the setup.py. That way users can just call `e_span <filepath> -T 300` (e.g. optional temperature flag with defaults). You should be able to make the switch pretty easily by looking at some of the other scripts in `bin/` (add_cns.py is a good example). All you need to do is convert this script to a single function with a few click decorators:

```python
@click.command('e_span', ...)
@click.argument('filepath')
@click.option('-T', ...)
def e_span(filepath, T=298.15):
    pass
```

Then there's a single line you need to add to the `console_scripts` section in the setup.py: `'e_span=canela.bin.e_span:e_span'`.

Using click will let python do a lot of the work for you to make the script easier to use and more accessible (adding -h/--help doc string and automatically adding the script to a user's path).

miuev committed:
e_span click-ify and import fixes

miuev commented:
OK, click-ified and updated the script, should work as you has suggested now. Had it like that before but it was hardcoded with arguments instead of the option flags. This is much nicer, thanks for teaching me about it.

Also, I noticed there was a circular import error with the `canela/bin/canela.py` file, so I renamed it to `canela_info.py` and changed the `setup.py` accordingly. Hope this is alright, the issue I saw was the `from canela import __version__` calls in the `bin` scripts. I think it saw the `bin/canela.py` and tried importing a version from there instead of the higher up `__version__` instantiated by `canela/_version.py`. If that seems not correct then we can discuss, I am honestly not 100% sure if that's what the issue was, but it seems to work OK now.

michael-cowan commented:
Great catch with the circular import error. Everything looks great to me - I'll merge it now
